Library/Homebrew/shims/super/cc

```diff
@@ -303,6 +303,7 @@ class Cmd
   end
 
   def ldflags_linux(args)
+    args.pop() unless tool =~ /^clang/
```

Comment:
A comment explaining why this isn't needed for clang would be good. Have you tested compiling with Clang on Linux and ensured this doesn't break things?

Reply:
@iMichka Do we currently have a way to use the `clang` superenv on Linux?  I'm not even sure if we officially support building with `clang` on Linux right now.  I'll update this with a comment to explain my understanding of why it is not needed, and I will also test `clang` if possible.